src/main/java/uk/gov/hmcts/divorce/notification/GetAllTemplatesFromNotify.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.hmcts.divorce.notification;
+
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.NotificationClientException;
+import uk.gov.service.notify.Template;
+import uk.gov.service.notify.TemplateList;
+
+import java.util.HashMap;
+import java.util.Map;
+
+// GetAllTemplatesFromNotify is a standalone util class which allows you to get all templates from Notify
+// which contain a given search criteria.
+public final class GetAllTemplatesFromNotify {
+
+    private static final String searchCriteria = "Enter search criteria here";
+    private static final String notificationType = "email";
+
+    private static final String apiKey = "";
+
+    private GetAllTemplatesFromNotify() {
+    }
+
+    public static void main(String[] args) {
+
+        NotificationClient client = getNotificationClient();
+
+        try {
+            TemplateList templates = client.getAllTemplates(notificationType);
+
+            System.out.println("Found " + templates.getTemplates().size() + " templates in total");
+
+            // Uncomment if searching template values.
+//            searchTemplateValues(templates);
+
+            // uncomment if searching template body contents.
+//            searchTemplateBody(templates);
+
+        } catch (NotificationClientException e) {
+            e.printStackTrace(System.out);
+        }
+    }
+
+    public static NotificationClient getNotificationClient() {
+        return new NotificationClient(apiKey);
+    }
+
+    private static void searchTemplateValues(TemplateList templates) {
+
+        int count = 0;
+
+        for (Template template: templates.getTemplates()) {
+
+            Map<String, Object> personalisation = template.getPersonalisation().orElse(new HashMap<>());
+
+            if (personalisation.containsKey(searchCriteria)) {
+                System.out.println("************************************************************");
+
+                if (template.getName().contains("COR") || template.getName().contains("NOT USED") || template.getName().contains("OLD")) {
```

Comment:
what's COR?

Reply:
Leftover from original code, my mistake. Removed and extracted the if statement code to a common func